.github/workflows/pull_request.yaml

```diff
@@ -1,10 +1,10 @@
-name: "PULL REQUEST"
+name: "Pull Request"
 
 on:
   pull_request:
     branches:
       - main
```

Comment:
does this mean we lint only on develop or on merge to main? is there ever a case where linting might break a PR on its way to main?

Reply:
This means we lint any PR where the target branch is develop or main. 99% of the time, develop should be the target. In the case of a hotfix, we might need to PR directly to main. In that case we would still want to lint that change.

This linting step is 100% risk-free because it does not fix linting issues. The lint step runs with the `--check` option, which will not make any changes, and the `--diff` option which shows a detailed output of the changes required for linting to pass.

Here's an example of the linting step failing: https://github.com/palmetto/palm-cli/runs/4385256425?check_suite_focus=true